db/data_migration/20150330081423_destroy_dependencies_for_superseded_editions.rb

```diff
@@ -0,0 +1,6 @@
+puts "Destroying dependencies for superseded editions"
+
+dependencies_for_superseded_editions = EditionDependency.includes(:edition).all.select {|ed| ed.edition.superseded? }
```

Comment:
quibble as it doesn't make any difference, but I don't think you need `all` here.